spec/support/examples_group/shared_examples_for_ext_management_system.rb

```diff
@@ -0,0 +1,40 @@
+shared_examples_for "ExtManagementSystem#pause!" do
+  before { Zone.seed }
+
+  it "disables the manager and moves it to the maintenance zone" do
+    ems.pause!
+
+    ems.reload
+
+    expect(ems.enabled).to eq(false)
+    expect(ems.zone).to eq(Zone.maintenance_zone)
+    expect(ems.zone_before_pause).to eq(zone)
+  end
+
+  it "disables all child managers and moves them to the maintenance zone" do
+    ems.pause!
+
+    ems.reload
+
+    ems.child_managers.each do |child_manager|
+      expect(child_manager.enabled).to eq(false)
+      expect(child_manager.zone).to eq(Zone.maintenance_zone)
+      expect(child_manager.zone_before_pause).to eq(zone)
+    end
+  end
+end
+
+shared_examples_for "ExtManagementSystem#resume!" do
+  before { Zone.seed }
+
+  it "resumes the manager and moves it to the original zone" do
+    ems.pause!
+    ems.reload
+    ems.resume!
+    ems.reload
```

Comment:
This is a little weird but the existing tests do this (https://github.com/ManageIQ/manageiq/pull/22716/files#diff-f50b4b351bf43b590f4afa70fdf2534bb4291339af5d57f9c283b73913a95ab0L623-L627)

It seemed difficult to setup the ems paused with `let(:ems)`.  Maybe we could add an Rspec trait for `:paused`, more investigation needed.